server_windows.go

```diff
@@ -1,8 +1,16 @@
+//go:build go1.18
```

Comment:
I am unsure that this go build tag is necessary. `x/sys/windows` should be compatible with at least some older versions?

Reply:
It was needed for using `any` - Now we should be ok without it